backend/common/src/node_types.rs

```diff
@@ -21,43 +38,70 @@ pub struct NodeDetails {
     pub startup_time: Option<Box<str>>,
 }
 
-#[derive(Deserialize, Debug, Clone, Copy, PartialEq, Eq, Default)]
+///
```

Comment:
no docs?

Reply:
I think I just added those to help visually separate the various types in that file better than they were.

I can def think about adding actual docs :D